packages/apps/autorest/CHANGELOG.md

```diff
@@ -1,22 +1,29 @@
-# AutoRest CLI 
+# Change Log - autorest
+
+This log was last generated on Thu, 04 Feb 2021 19:16:37 GMT and should not be manually modified.
+
+## 3.0.6336
+Thu, 04 Feb 2021 19:05:18 GMT
+
+### Patches
+
+- Consolidate tsconfig settings with other projects
+- Internal code linting fixes
+
+## 3.0.x
```

Comment:
Interesting, is this migrated changelog entries for previous versions?

Reply:
Yeah, I manually added entires from the previous changelogs